kernel/entries.ml

```diff
@@ -87,9 +87,16 @@ type inline = int option (* inlining level, None for no inlining *)
 type parameter_entry = 
     Constr.named_context option * types in_constant_universes_entry * inline
 
+type primitive_entry = {
+  prim_entry_type : types option;
```

Comment:
Are there actually cases where you set this to a non-empty type? All the examples from the stdlib seem not to provide a type.

Reply:
Well, the syntax is there, right? `Primitive foo : XX := #YY`.

Reply:
Indeed, but if all instances are not using it, is it worth it?

Reply:
I think it is more uniform with `Definition` and co, and the fact that it is not used in the stdlib today does not imply much. Especially, the cost of this is very low.

Reply:
Fine, in that case, you can dismiss the comment about the constraints below.